src/js/helpers/validator.js

```diff
@@ -4,7 +4,7 @@
  * @returns Boolean
  */
 export const isEmail = email => {
-  const re = /^(([^<>()[\].,;:\s@"]+(\.[^<>()[\].,;:\s@\"]+)*)|(\".+\"))@(([[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+  const re = /^(([^<>()[\].,;:\s@"]+(\.[^<>()[\].,;:\s@"]+)*)|(".+"))@(([[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
The email validator passes this email -> retro@rabbit.com.com.com
That doesn't sound like a valid email.